hoot-core/src/main/cpp/hoot/core/conflate/UnifyingConflator.cpp

```diff
@@ -299,14 +295,41 @@ void UnifyingConflator::apply(OsmMapPtr& map)
   _stats.append(SingleStat("Apply Post Ops Time (sec)", timer.getElapsedAndRestart()));
 }
 
+void UnifyingConflator::_mapUnknownIdsBackToModifiedElements(OsmMapPtr& map)
+{
+  for (size_t i = 0; i < _mergers.size(); ++i)
+  {
+    set< pair<ElementId, ElementId> > impactedUnknown1ElementIds =
+      _mergers[i]->getImpactedUnknown1ElementIds();
+    for (set< pair<ElementId, ElementId> >::const_iterator it = impactedUnknown1ElementIds.begin();
+         it != impactedUnknown1ElementIds.end(); ++it)
+    {
+      ElementId eid1 = it->first;
+      ElementId eid2 = it->second;
+      LOG_VARD(eid1);
+      LOG_VARD(eid2);
+
+      if (eid1.getType() == eid2.getType())
+      {
+        LOG_DEBUG("Setting " << eid1.getId() << " on " << eid2.getId() << "...");
+        ElementPtr replacementElement = map->getElement(eid2);
+        LOG_VARD(replacementElement->getElementId().getType());
+        ElementPtr newUnknown1Element(replacementElement->clone());
+        newUnknown1Element->setId(eid1.getId());
+        //TODO: I believe this should be status=3, but I'm not 100% convinced yet.
+        newUnknown1Element->setStatus(Status::Conflated/*Status::Unknown1*/);
```

Comment:
Is this the code that picks one feature from the split results to retain the original element id so a modify step will occur in the changeset?
If so I'm not sure a merge will always happen on this element, so the status could be either Conflated or Unknown1.  Is there a way to tell if this element was merged with a feature from the secondary input?